cluster/addons/dns/kube2sky/kube2sky.go

```diff
@@ -195,7 +195,7 @@ func newKubeClient() (*kclient.Client, error) {
 		var err error
 		if config, err = kclientcmd.NewNonInteractiveDeferredLoadingClientConfig(
 			&kclientcmd.ClientConfigLoadingRules{ExplicitPath: *argKubecfgFile},
-			&kclientcmd.ConfigOverrides{ClusterInfo: kclientcmdapi.Cluster{Server: masterUrl, InsecureSkipTLSVerify: true}}).ClientConfig(); err != nil {
+			&kclientcmd.ConfigOverrides{ClusterInfo: kclientcmdapi.Cluster{Server: masterUrl}}).ClientConfig(); err != nil {
```

Comment:
I'd be happy if the process was using the server value from the kubeconfig file. But if you actually look at how these processes run, they override the server location with the value of the `--kube_master_url` flag which defaults to `https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT}` (which turns into `https://10.0.0.2:443`). I want the process to use the value from the kubeconfig file instead.